src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -591,6 +600,31 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
         return api.getUsersAPI(false).getAvatarUrlWithAccessToken(userId, file.id);
     };
 
+    handleAnnotationSelect = (annotationId: string, { fileVersionId, location }: AnnotationSelectMeta): void => {
```

Comment:
Do we already have something like this elsewhere? I see a simliar handler in one of the other Sidebar components. Can/should we combine them to avoid sprawl for this code path?

Reply:
Sure let me see how we can consolidate this